src/app/admin/admin-reports/filtered-items/filtered-items.component.ts

```diff
@@ -0,0 +1,301 @@
+                            this.collections.push(collVO);
+                          });
+                    }
+                  );
+            });
+          }
+        );
+  }
+
+  loadPresetQueries(): void {
+    this.presetQueries = [
+      PresetQuery.of('new', 'admin.reports.items.preset.new', []),
+      PresetQuery.of('q1', 'admin.reports.items.preset.hasNoTitle', [
+        QueryPredicate.of('dc.title', QueryPredicate.DOES_NOT_EXIST)
+      ]),
+      PresetQuery.of('q2', 'admin.reports.items.preset.hasNoIdentifierUri', [
+        QueryPredicate.of('dc.identifier.uri', QueryPredicate.DOES_NOT_EXIST)
+      ]),
+      PresetQuery.of('q3', 'admin.reports.items.preset.hasCompoundSubject', [
+        QueryPredicate.of('dc.subject.*', QueryPredicate.LIKE, '%;%')
+      ]),
+      PresetQuery.of('q4', 'admin.reports.items.preset.hasCompoundAuthor', [
+        QueryPredicate.of('dc.contributor.author', QueryPredicate.LIKE, '% and %')
+      ]),
+      PresetQuery.of('q5', 'admin.reports.items.preset.hasCompoundCreator', [
+        QueryPredicate.of('dc.creator', QueryPredicate.LIKE, '% and %')
+      ]),
+      PresetQuery.of('q6', 'admin.reports.items.preset.hasUrlInDescription', [
+        QueryPredicate.of('dc.description', QueryPredicate.MATCHES, '^.*(http://|https://|mailto:).*$')
+      ]),
+      PresetQuery.of('q7', 'admin.reports.items.preset.hasFullTextInProvenance', [
+        QueryPredicate.of('dc.description.provenance', QueryPredicate.MATCHES, '^.*No\. of bitstreams(.|\r|\n|\r\n)*\.(PDF|pdf|DOC|doc|PPT|ppt|DOCX|docx|PPTX|pptx).*$')
+      ]),
+      PresetQuery.of('q8', 'admin.reports.items.preset.hasNonFullTextInProvenance', [
+        QueryPredicate.of('dc.description.provenance', QueryPredicate.DOES_NOT_MATCH, '^.*No\. of bitstreams(.|\r|\n|\r\n)*\.(PDF|pdf|DOC|doc|PPT|ppt|DOCX|docx|PPTX|pptx).*$')
+      ]),
+      PresetQuery.of('q9', 'admin.reports.items.preset.hasEmptyMetadata', [
+        QueryPredicate.of('*', QueryPredicate.MATCHES, '^\s*$')
+      ]),
+      PresetQuery.of('q10', 'admin.reports.items.preset.hasUnbreakingDataInDescription', [
+        QueryPredicate.of('dc.description.*', QueryPredicate.MATCHES, '^.*[^\s]{50,}.*$')
+      ]),
+      PresetQuery.of('q12', 'admin.reports.items.preset.hasXmlEntityInMetadata', [
+        QueryPredicate.of('*', QueryPredicate.MATCHES, '^.*&#.*$')
+      ]),
+      PresetQuery.of('q13', 'admin.reports.items.preset.hasNonAsciiCharInMetadata', [
+        QueryPredicate.of('*', QueryPredicate.MATCHES, '^.*[^[:ascii:]].*$')
+      ])
+    ];
+  }
+
+  loadMetadataFields(): void {
+    this.metadataFields = [];
+    this.metadataFieldsWithAny = [];
+    let anyField$ = this.translateService.stream('admin.reports.items.anyField');
+    this.metadataFieldsWithAny.push(OptionVO.itemLoc('*', anyField$));
+    this.metadataSchemaService.findAll({ elementsPerPage: 10000, currentPage: 1 }).pipe(
```

Comment:
Same here, huge page-sizes should be avoided. In this case you can reuse [`ds-metadata-field-selector`](https://github.com/DSpace/dspace-angular/blob/829b111663111fbcf1477fc7115417912cd06584/src/app/dso-shared/dso-edit-metadata/metadata-field-selector/metadata-field-selector.component.ts#L36) to get a paginated dropdown of metadata fields

Reply:
Same comment as above.